airdrop/infrastructure/models.py

```diff
@@ -164,6 +164,7 @@ class ClaimHistory(Base, AuditClass):
         nullable=False,
     )
     address = Column("address", VARCHAR(50), nullable=False, index=True)
+    type = Column("type", VARCHAR(50), nullable=True)
```

Comment:
column name type is very generic. can we have a more meaningful column name here?